shared/src/data/default-state/patient-templates.ts

```diff
@@ -12,173 +13,1210 @@ const defaultMaleImage: ImageProperties = {
+    [
+        {
+            matchingHealthStateId: yellowStartPhase4RSDecisionState.id,
+            earliestTime: 4 * phaseTime,
+        },
+    ]
+);
+
+const yellowStartPhase9RADecisionState = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.blackMax,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.blackMax,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.blackMax,
+            1
+        ),
+        0
+    ),
+    [
+        {
+            matchingHealthStateId: noChangesState.id,
+            minimumHealth: healthPointsDefaults.yellowAverage,
+        },
+        {
+            matchingHealthStateId: noChangesState.id,
+            maximumHealth: healthPointsDefaults.blackMax,
+        },
+    ]
+);
+
+const yellowStartPhase8RADecisionState = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        0
+    ),
+    [
+        {
+            matchingHealthStateId: yellowStartPhase9RADecisionState.id,
+            earliestTime: phaseTime,
+        },
+    ]
+);
+
+const yellowStartPhase7RSDecisionState = PatientHealthState.create(
+    FunctionParameters.create(
+        calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        ),
+        -calculateHealthChange(
+            healthPointsDefaults.yellowAverage,
+            healthPointsDefaults.redAverage,
+            1
+        )
+    ),
+    [
+        {
+            matchingHealthStateId: yellowStartPhase8RADecisionState.id,
+            earliestTime: phaseTime,
+        },
+    ]
+);
+
+const yellowFor3PhasesState = PatientHealthState.create(
+    FunctionParameters.create(0, 0, 0, 0),
+    [
+        {
+            matchingHealthStateId: yellowStartPhase7RSDecisionState.id,
+            earliestTime: 3 * phaseTime,
+        },
+    ]
+);
+
+const redFor2PhasesState = PatientHealthState.create(
```

Comment:
This name doesn't match the behavior (at least the naming scheme is not consistent: above, the same scheme means that there is no change, here there is change). I think a name alongside `redUntilPhase2State` would be better.

Reply:
has been changed to `redUntilYellowFor2PhasesState`